flight/hal/cubesat.py

```diff
@@ -190,6 +171,13 @@ def IMU(self):
         """
         return self.__imu
 
+    @property
+    def IMU_NAME(self):
+        """IMU_NAME: Returns the IMU name
+        :return: object or None
+        """
+        return self.__imu_name
+

```

Comment:
WHen you modify the HAL, think about also modifying the emulation side to match it, else you break it (as is the case now)